OSDOCS-17165-backup-etcd CQA (#117074)
openshift/openshift-docs: modules/backup-etcd.adoc

= Backing up etcd data

[role="_abstract"]

> You can back up etcd data by creating an etcd snapshot and saving the static pod resources on a control plane host. This backup preserves cluster state so you can restore etcd after a failure or before a cluster update.

Reviewer: I have a concern with this short description--namely the second sentence, specifically ". . . preserves cluster state so you can restore etcd after a failure or before a cluster update.". 

My issue is that this information isn't covered in this module, or anywhere in these docs, so I think it's new. Because it's new, it would probably need to be verified by QE; that is, I'm not sure if you'd restore etcd "after a failure or before a cluster update", and you'd want to get that information verified.

A safer way would probably be something like...

```suggestion
You can back up etcd data by creating an etcd snapshot and saving the static pod resources on a control plane host. This backup preserves the cluster state and provides the resources required to restore etcd at a later time.
```

I *prefer* your version because it answers WHY they'd do that. But unless you know QE/SME that could quickly verify, or a writer who could, or you could find this info somewhere within one of these modules, more vague let's you avoid QE for now.

Author: Ok, sounds good. I'm using your rewrite updates but also asking Cursor to redo them based on extractive rather than abstractive. I did that to the one above and it came pretty close to the updates that you made. From now on I'll do extractive rather than abstractive. Thanks for the tip
